writers/file_test.go

```diff
@@ -67,7 +67,7 @@ func TestInvalidPath(t *testing.T) {
 
 // getTempFilePath returns path to temp file and cleans it up after test finishes
 func getTempFilePath(t *testing.T, name string) (string, error) {
-	if name != "" {
+	if name == "" {
 		name = "file.txt"
 	}
 	path, err := filepath.Abs("../tmp/" + name)
```

Comment:
Just poping out in your PR, but why having such method and not using

https://pkg.go.dev/testing#T.TempDir

Reply:
Yeah, TempDir works great here. Thanks!